### ISSUE TITLE
Issue labels

### ISSUE THREAD

**Issue by scottdorman** [dotnet/vblang#17]
For consistency, we should keep the issue labels in both repos the same. This would include both names and colors. Right now, they're almost the same:

### C# (https://github.com/dotnet/csharplang/labels)
- `Meeting notes issue`
- `Proposal champion`
- `Proposal issue`
- `Spec defect`

### VB (https://github.com/dotnet/vblang/labels)
- `Design-Meeting-Notes`
- `Proposal`
- `Scenario`
- `Spec-Defect`

![2017-02-10 08_52_10-labels dotnet_csharplang - microsoft edge](https://cloud.githubusercontent.com/assets/2532850/22829055/837f4850-ef6e-11e6-8cdb-b6e5cfbff35a.png)

**Comment by scottdorman** [comment]
If we want additional tags, @AdamSpeight2008's [proposed list](https://github.com/dotnet/vblang/issues/8) is a good start.

**Comment by AnthonyDGreen** [comment]
I don't believe in spaces on the web. It always finds its way into an ugly URL. Mads is less sensitive to it because he doesn't use SharePoint as much as I do :P

**Comment by AdamSpeight2008** [comment]
Hyphenated doesn't require the tag to be in quotes when search for them on GitHub

**Comment by scottdorman** [comment]
@AdamSpeight2008 I think @AnthonyDGreen is in agreement - spaces are bad. :) I agree. The spaces "look" better visually but create ugly URLs and require them to be quoted when searching in GitHub. 

No matter what, I think the labels across both repositories should be consistent - both in name, spelling, and hyphenation. (Personally, I like them all lower case with hyphens, but that's just me.) We just need to get @MadsTorgersen and @AnthonyDGreen (and probably @gafter as well, since it seems he's the one who's been assigning most of the labels) to agree on a common set of labels and spelling. 

We're almost there now, so I think with a few minor tweaks we can get there:

* `Discussion`
* `Feature-Request`
* `Meeting-Notes`
* `Proposal-Champion`
* `Proposal`
* `Spec-Defect`
* `Scenario` (? - I'm not sure what this would be used for)
